framework/PageForm/PageForm.tsx

```diff
@@ -17,16 +16,26 @@ import {
   Path,
   useForm,
   UseFormReturn,
-  useFormState,
 } from 'react-hook-form';
-import { useTranslation } from 'react-i18next';
 import { Scrollable } from '../components/Scrollable';
 import { useBreakpoint } from '../components/useBreakPoint';
 import { SettingsContext } from '../Settings';
 import { useFrameworkTranslations } from '../useFrameworkTranslations';
 import { ErrorAlert } from './ErrorAlert';
 import { genericErrorAdapter } from './genericErrorAdapter';
 import { ErrorAdapter } from './typesErrorAdapter';
+import { PageFormSubmitButton, PageFormCancelButton } from './PageFormButtons';
+import styled from 'styled-components';
+
+const FormContainer = styled(PageSection)`
+  margin-block-end: var(--pf-v5-global--spacer--xl);
+`;
+
+const FormActionGroup = styled(ActionGroup)`
+  && {
+    margin-block-start: unset;
+  }
+`;
 
 export interface PageFormProps<T extends object> {
```

Comment:
Could we extend GenericFormProps here? Since they have so many common properties.

Reply:
I don't think they need to share an interface just because they happen to be similar. I kind of prefer seeing the whole of PageFormProps listed out in one place